applications/CompressiblePotentialFlowApplication/custom_processes/define_3d_wake_process.h

```diff
@@ -90,27 +89,36 @@ class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) Define3DWakeProcess
     ///@name Member Variables
     ///@{
 
-    // The airfoil model part containing the trailing edge
+    // The model part containing the trailing edge
     ModelPart& mrTrailingEdgeModelPart;
     ModelPart& mrBodyModelPart;
-    ModelPart& mrStlWakeModelPart;
     // Tolerance to avoid nodes laying exactly on the wake
     double mTolerance;
     BoundedVector<double, 3> mWakeNormal;
     BoundedVector<double, 3> mWakeDirection;
     BoundedVector<double, 3> mSpanDirection;
 
-    bool mSwitchWakeDirection;
-    bool mCountElementsNumber;
-    bool mWriteElementsIdsToFile;
     bool mShedWakeFromTrailingEdge;
-    bool mDecreaseWakeWidthAtTheWingTips;
+    bool mVisualizeWakeVTK;
     int mEchoLevel;
-
-    double mSheddedWakeDistance;
-    double mSheddedWakeElementSize;
-
-    BoundedVector<double, 3> mWakeNormalOld;
+    
+    double mShedWakeLength;
+    double mShedWakeElementSize;
+    double mShedGrowFactor;
+    double mShedProjectionRootEdge;
+
+    BoundedVector<double, 3> mWakedrTraslation;
+
+    std::string mBluntTESurfaceModelPartName;
+    std::string mUpperSurfaceModelPartName;
+    std::string mLowerSurfaceModelPartName;
+    std::string mRootPointsModelPartName;
+    std::string mTipPointsModelPartName;
+    std::string mTailModelPartName;
+    
+    std::unordered_set<IndexType> mBluntIds; 
```

Comment:
Do we need to save them? Can't we just use it and delete them? In other words, do we need these during the entire simulation?

If we really need to keep them, can't we use a flags? (even a local one).